playbooks/app_dev/builds.adoc

```diff
@@ -335,9 +337,13 @@ There are a number of existing S2I builders that you can look to leverage for yo
 
 === Binary deployments
 
-In certain cases, an application may be previously compiled outside of an OpenShift build. You can use the _assemble_ phase of an S2I build process to retrieve the previously packaged artifact and insert it into the typical build process of the builder being leveraged.
 
-The following is an example of a deployment of a binary application for the OpenShift JBoss EAP image
+In certain cases, an application may be previously compiled outside of an OpenShift build. There are two methods to include an existing binary artifact(s) in the resulting image:
+
+1. Customize the _assemble_ script as part of the S2I build process to retrieve a previously packaged artifact from a remote source (such as an artifact repository) and place in the JBoss deployments folder.
```

Comment:
Both of these methods seems specific to JBoss. Although that is most likely capturing the majority of our customers and engagements, could it be written a bit more generic? ... or potentially prefaced with a comment stating that this is JBoss specific, and a similar approach can also be used with other types of deployment?